### PR TITLE
chore: update synthconfig to include generation of README.md

### PR DESCRIPTION
**Pull request by BenWhitehead**

### CODE REVIEW

File: .readme-partials.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+custom_content: |
+  Cloud Firestore is a fully-managed NoSQL document database for mobile, web,
+  and server development from Firebase and Google Cloud Platform.  It's backed
+  by a multi-region replicated database that ensures once data is committed,
+  it's durable even in the face of unexpected disasters. Not only that,
+  but despite being a distributed database, it's also strongly consistent and
+  offers seamless integration with other Firebase and Google Cloud Platform
+  products, including Google Cloud Functions.
```
Comment (chingor13) [inline .readme-partials.yaml:8]: You may want to put this in the .repo-metadata.json in the `api_description` field starting with "is a fully-managed..."

File: README.md [hunk 2]
```diff
@@ -48,58 +55,109 @@ libraryDependencies += "com.google.cloud" % "google-cloud-firestore" % "1.35.2"
...
+You will need a [Google Cloud Platform Console][developer-console] project with the Cloud Firestore [API enabled][enable-api].
 
-To get help, follow the instructions in the [shared Troubleshooting document](https://github.com/googleapis/google-cloud-common/blob/master/troubleshooting/readme.md#troubleshooting).
+[Follow these instructions][create-project] to get your project set up. You will also need to set up the local development environment by
+[installing the Google Cloud SDK][cloud-sdk] and running the following commands in command line:
+`gcloud auth login` and `gcloud config set project [YOUR PROJECT ID]`.
 
-Transport
----------
-Firestore uses gRPC for the transport layer.
```
Comment (chingor13) [inline README.md:70]: There's a `transport` field you can set to `grpc` in the `.repo-metadata.json` if you want to keep this section.

File: .readme-partials.yaml (hunk 1 above)
Comment (BenWhitehead) [reply .readme-partials.yaml]: Done

File: README.md (hunk 2 above)
Comment (BenWhitehead) [reply README.md:70]: Done

**Pull request merged**